src/analysis/special_functions/log/base.lean

```diff
@@ -52,6 +52,16 @@ by simp_rw [logb, log_div hx hy, sub_div]
 
 @[simp] lemma logb_inv (x : ℝ) : logb b (x⁻¹) = -logb b x := by simp [logb, neg_div]
 
+@[simp] lemma inv_logb {a b : ℝ} : (logb a b)⁻¹ = logb b a := by simp_rw [logb, inv_div]
```

Comment:
It might also be nice to have the lemma that `logb a b * logb b c = logb a c` and `logb a c / logb b c = logb a b`, both of which I would expect to be easy to prove.

Reply:
Oh, also
```suggestion
@[simp] lemma inv_logb (a b : ℝ) : (logb a b)⁻¹ = logb b a := by simp_rw [logb, inv_div]
```

Reply:
I also added those suggestions

The two additional lemmas you suggested are slightly less general than I hoped because `log x / log x = 1` does not work if `x = 1`, `x = 0`, or `x = -1`